internal/testutil/mig/provider.go

```diff
@@ -11,6 +11,7 @@ import (
 
 func SkipIfVersionBelow(tb testing.TB, minVersion string) {
 	tb.Helper()
+	validateConflictingEnvVars(tb)
```

Comment:
do we need to call it here?   I assume that checkLastVersion will be called at some point in a mig test

Reply:
I found some examples where only `acc` pre checks are used, e.g., [here.](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/service/streaminstance/resource_stream_instance_migration_test.go#L19)

Reply:
oh i see, so we have some inconsistencies as we should call checkLastVersion before calling versionConstraint, but it's not happening in the example you're mentioning.

mig.ExternalProviders calls versionConstraint but checkLastVersion is not called because the test is missing a mig precheck

Reply:
all mig tests should call a mig precheck but not sure if it's easy to enforce

Reply:
Yes, it would be nice if we could enforce it.
For now, I pefer keeping the `validateConflictingEnvVars` in the `SkipIfVersionBelow` 👍